main.css

```diff
@@ -95,42 +95,48 @@ body {
     display: inline-block;
   }
 
-  .prog-table {
-    border-collapse: collapse;
-    margin: 1rem;
+.prog-table {
+  border-collapse: collapse;
+  margin: 1rem;
+}
+  .prog-table__row {
+    -webkit-box-shadow: inset 0px 1px 0px 0px #C0C0C2;
+    -moz-box-shadow: inset 0px 1px 0px 0px #C0C0C2;
+    box-shadow: inset 0px 1px 0px 0px #C0C0C2;
+  }
+  .prog-table__header {
+    padding: 1rem 0.25rem;
+    text-align: left;
+    font-family: 'Raleway', sans-serif;
+  }
+      .prog-table__header--datetime {
+        padding: 1rem;
+      }
+  .prog-table__time {
+    display: block;
+    font-size: 0.875rem; /* 14px */
+    margin-bottom: 0.5em; /* 7px */
+  }
+  .prog-table__date {
+    font-size: 0.875rem; /* 14px */
+    font-weight: 600;
   }
-    .prog-table__row {
-      -webkit-box-shadow: inset 0px 1px 0px 0px #C0C0C2;
-      -moz-box-shadow: inset 0px 1px 0px 0px #C0C0C2;
-      box-shadow: inset 0px 1px 0px 0px #C0C0C2;
-    }
-    .prog-table__header {
-      padding: 1rem 0.25rem;
-      text-align: left;
-      font-family: 'Raleway', sans-serif;
-    }
-        .prog-table__header--datetime {
-          padding: 1rem;
-        }
-    .prog-table__time {
-      display: block;
-      font-size: 0.875rem; /* 14px */
-      margin-bottom: 0.5em; /* 7px */
-    }
-    .prog-table__date {
-      font-size: 0.875rem; /* 14px */
-      font-weight: 600;
-    }
 
-.prog-bar{}
-  .prog-bar__button{
-    margin: 0.75rem 1.5rem;
+.programme-controls{
+  border-top: solid 1px rgb(192, 192, 194);
+  border-bottom: solid 1px rgb(192, 192, 194);
+  padding: 0 2rem;
+}
+  .prog-controls__dividing-section{
+    display:inline-block;
+    padding: 0.625rem 1rem;
```

Comment:
Sorry, I gave you some bogus advice, the size of the padding should be as follows to account for the height of the buttons content area being an odd number
```css
padding: 0.6875rem 1rem 0.625rem; /* 11px 16px 10px */
```